tests/infrastructure/golden_images/update_boot_source/test_boot_sources_vm.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+import pytest
+
+from utilities.infra import assert_os_version_mismatch_in_vm, validate_os_info_vmi_vs_linux_os
+from utilities.storage import data_volume_template_with_source_ref_dict
+from utilities.virt import VirtualMachineForTests, running_vm
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def auto_update_boot_source_instance_type_vm(
+    unprivileged_client,
+    namespace,
+    modern_cpu_for_migration,
```

Comment:
Are you migrating a VM in this test?
Don't we need modern_cpu only for Windows VMs?

Reply:
RHEL-10 needs modern CPU

Reply:
@RoniKishner don't set `cpu_model` explicitly if you are not goint to migrate the VM.

By default the host-model will be used (which is pointed to the most modern cpu)